src/utils/url.ts

```diff
@@ -1,9 +1,11 @@
-export function formatUrl(apiKey : string, endpoint : string[], query : Record<string, any> = {}) : string {
-	let url = endpoint
+import { APIKey } from '../types';
+
+export const formatUrl = (apiKey : APIKey, endpoint : string[], query : Record<string, any> = {}) : string => {
```

Comment:
Is there a particular reason you made this an arrow function? If not, I think it would be clearer if you change this back to a regular function notation.

Reply:
All other functions I've changed are arrow functions, so this would make it consistent